### PR TITLE
Run cljfmt on edn files

### PR DESCRIPTION
**Pull request by asponring**
Companion to https://github.com/cespare/goclj/pull/96. For vim users, format edn files on save just like clj files.

### CODE REVIEW

**Comment by asponring** [comment]
@cespare PTAL.

File: README.md [hunk 1]
```diff
@@ -20,5 +20,5 @@ Install vim-cljfmt with your method of choice.
 ## Settings
 
 - `g:cljfmt_on_save` controls whether cljfmt automatically calls `:CljFmt` when
-  saving a .clj or .cljs buffer. This is set to 1 by default.
+  saving a .clj, .cljs or .edn buffer. This is set to 1 by default.
```
Comment (cespare) [inline README.md]: Please add an oxford comma (here and below).

**Review by cespare — APPROVED** [review]
Could you also please add .cljc files while you're here? The only reason I didn't include it back in 31c1338576004addb9aedc80ed076f8bd4274440 was due to cljfmt not handling reader conditionals at the time but I fixed that long ago.

**Comment by asponring** [comment]
Addressed CR. I believe I don't have permission to merge in this repo, so I'll let you know when it's ready to go.

**Comment by cespare** [comment]
Thanks!

**Pull request merged**